calour/heatmap/heatmap.py

```diff
@@ -86,10 +86,55 @@ def _truncate_middle(x, length=16):
         return ['%s..%s' % (i[:mid], i[-mid:]) if len(i) > length else i for i in y]
 
 
+def _set_axis_ticks(ax, which, ticklabels, tickmax, n, kwargs, ticklabel_len):
+    ''''''
```

Comment:
i think it's worth having a (small) description & parameter list for this function? otherwise in a few years we will forget what it does and how :)
i.e. what is 'which'

Reply:
added